### PR TITLE
Ports+LibC+LibM: Make ScummVM run games

### PR DESCRIPTION
**Pull request by AnicJov**
By adding these libs we make ScummVM capable of running a lot of games :)
There are still a few patches to the actual scummvm port that need to be made, which I will submit in a separate PR (#6134).

Hooray for Myst on Serenity!
image

### CODE REVIEW

File: Ports/faad2/package.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=faad2
+version=2_10_0
+useconfigure=true
+files="https://github.com/knik0/faad2/archive/refs/tags/${version}.tar.gz faad2-${version}.tar.gz"
+
+post_fetch() {
+    cd "faad2-${version}"
+    autoreconf -vif
+    cd ..
```
Comment (linusg) [inline Ports/faad2/package.sh:10]: That's exactly what `run` does :)

https://github.com/SerenityOS/serenity/blob/15a820e9ad47fa61b6010fac8dbd58c6ea853f81/Ports/.port_include.sh#L41-L44

```suggestion
    run autoreconf -vif
```

File: Ports/fribidi/package.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+port=fribidi
+version=1.0.10
+useconfigure=true
+files="https://github.com/fribidi/fribidi/releases/download/v${version}/fribidi-${version}.tar.xz fribidi-${version}.tar.xz"
+makeopts="CFLAGS=-std=c99"
+
+pre_configure() {
+    cd "fribidi-${version}"
+    aclocal
+    cd ..
```
Comment (linusg) [inline Ports/fribidi/package.sh:11]:
```suggestion
    run aclocal
```

File: Ports/readline/package.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=readline
+version=8.1
+useconfigure=true
+files="ftp://ftp.gnu.org/gnu/readline/readline-${version}.tar.gz readline-${version}.tar.gz"
```
Comment (linusg) [inline Ports/readline/package.sh:5]: Sweet, I'll update the Python port to build with readline which should make the REPL experience a lot better :D

File: Userland/Libraries/LibC/sys/timeb.cpp [hunk 4]
```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021, Andreas Kling <kling@serenityos.org>
```
Comment (linusg) [inline Userland/Libraries/LibC/sys/timeb.cpp:2]: This file was written by you!

File: Userland/Libraries/LibC/sys/timeb.h [hunk 5]
```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021, Andreas Kling <kling@serenityos.org>
```
Comment (linusg) [inline Userland/Libraries/LibC/sys/timeb.h:2]: same here

**Review by linusg — CHANGES_REQUESTED** [review]

File: Ports/faad2/package.sh (hunk 1 above)
Comment (AnicJov) [reply Ports/faad2/package.sh:10]: Did not know this! Will fix

**Comment by awesomekling** [comment]
The "Ports: Add flac" commit is full of unrelated changes.

**Comment by GMTA** [comment]
@AnicJov this is a pretty big PR; why do the datafiles from a game released in 1993 require flac and libtheora?

**Pull request closed without merging**